cabal.project

```diff
@@ -55,95 +55,109 @@ package contra-tracer
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/iohk-monitoring-framework
-  subdir:   contra-tracer
   tag: dd30455144e11efb435619383ba84ce02aee720d
+  --sha256: 1g08bg99fvss99kg27l7pmxm7lh60573xln8l8x2rzvwfvfgk2i5
```

Comment:
just out of curiosity, what are these commented out `sha256` fields?

Reply:
IIRC, `haskell.nix` requires them but cabal can't yet use them.